projects.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: page
+title: "GitHub Projects"
+css: ["projects.css"]
+js: ["https://cdnjs.cloudflare.com/ajax/libs/geopattern/1.2.3/js/geopattern.min.js", "projects.js", "projectsLogo.js"]
+---
+{% include projects.html %}
```

Comment:
Is this auto generated based on the contents of our GH organisation? Are there any options for how these are ordered? (would be nice to do it either by activity or number of stars if that was possible)

Reply:
Would also be good to only include "source" projects (i.e. not ones that we've forked from other places - I see a slider at the top of the page which is supposed to control this, you have to toggle it on and off to work so it seems like it's initial state is wrong?)

Reply:
Good questions.
For the first that's correct, this is auto-generated using GitHub APIs. 
In fact, the list is sorted based on number of stars. Code for that is [here](https://github.com/HotelsDotCom/HotelsDotCom.github.io/blob/gh-pages/_includes/projects.html#L1)) which I believe works for us?

For the second question, good catch, I introduced a bug when changing this functionality. 
I've now fixed it [here](https://github.com/HotelsDotCom/HotelsDotCom.github.io/blob/gh-pages/_includes/projects.html#L5), and [here](https://github.com/HotelsDotCom/HotelsDotCom.github.io/blob/gh-pages/_includes/projects.html#L10). 
Let me know if this looks ok.